s4e-web/src/app/views/settings/people/person-form/person-form.component.html

```diff
@@ -1,34 +1,37 @@
-<header class="page__header">
-  <h1 i18n>Dodaj nową osobę</h1>
-</header>
 
-<section class="page__content">
-  <form [formGroup]="form" class="page__form">
-    <ul>
-      <li>
-        <label for="name" i18n>Imię</label>
-        <input id="name" type=text formControlName="name">
-      </li>
-      <li>
-        <label for="surname" i18n>Nazwisko</label>
-        <input id="surname" type=text formControlName="surname">
-      </li>
-      <li>
-        <label for="email" i18n>Email</label>
-        <input id="email" type="email" formControlName="email">
-      </li>
-      <li>
-        <strong i18n>Grupy</strong>
-        <s4e-check-box-list [items]="groups$ | async" formControlName="groupSlugs"></s4e-check-box-list>
-        <ng-template #groupsLoading>
-          Ładowanie...
-        </ng-template>
-      </li>
-      <li class="actions">
-        <button type="submit" (click)="saveAndNext()" [disabled]="loading$ | async" class="button button--primary" i18n>Zapisz i dodaj kolejną osobę</button>
-        <button type="submit" (click)="saveAndBack()" [disabled]="loading$ | async" class="button button--primary" i18n>Zapisz i zamknij stronę</button>
-        <a routerLink=".." queryParamsHandling="preserve" i18n>Anuluj</a>
-      </li>
-    </ul>
-  </form>
-</section>
+
+<s4e-generic-modal [buttonX]="true" [modalId]="modalId">
+  <div class="s4e-modal-header" i18n>{{ !!person ? 'Edytuj' : 'Dodaj' }} użytkownika</div>
+  <div class="s4e-modal-body">
+    <section class="page__content">
+      <form [formGroup]="form" class="page__form">
+        <ul>
+          <li>
+            <ext-input formControlName="name" label="Imię" i18n></ext-input>
+          </li>
+          <li>
+            <ext-input formControlName="surname" label="Nazwisko" i18n></ext-input>
+          </li>
+          <li>
+            <ext-input type="email" formControlName="email" label="Email" i18n></ext-input>
```

Comment:
If `!!person` then these inputs should be disabled, as you'd be editing an existing user.
Also, I noticed when I edit these that `POST /institutions/{institution}/users` request is sent. This should actually result in some error on the backend side, which it doesn't (that's an issue on backend side, I added its description [here](https://github.com/cyfronet-fid/sat4envi/issues/517#issuecomment-644236048)), but front should instead send `PUT /institutions/{institution}/users`

Reply:
I've updated fields to be disabled in edition mode and POST has been changed to PUT.
In case of an invalid payload that is sending with the form to API while edition, I'll just leave it until we'll have any conception of people and group management.